pages/views/roles/data.md

```diff
@@ -0,0 +1,80 @@
+
+### Direct
+{% include categoryLabel.html label="CatO" %}
+{% include categoryItem.html item="CatONetworkDevices" id="NetworkDevices" %}
+
+Transferring data over networks incurs an electricity cost, and therefore a carbon cost. 
+
+{% include categoryItem.html item="CatOServers" id="Servers" %}
+
+When hosting locally, the biggest factor in emissions is likely to be the embodied carbon of the servers.
+
+| Storage Type            | Embedded Carbon (kgCO2e/TB) | Operating Power (W/TB) |
+| ----------------------- | --------------------------: | ---------------------: |
+| Hard Disk Drive (HDD)   | 38                          | 1                      |
+| Solid State Drive (SSD) | 316                         | 2.5                    |
+
+(These figures are for a 8 TB HDD and a 1 TB SSD. Of course, there are reasons for using SSDs over HDDs, such as faster read/write speed.)
+
+A HDD will therefore have roughly equal amortised embodied carbon and operating emissions over 10 years of constant operation (while a SSD would require 35 years!). Since most storage media have a typical use life of around 5 years, increasing the lifespan of storage media is likely to have a significant effect on average emissions. 
+
+#### Long-term Copies and Backups
+
+Backups are commonly recommended for most data, and certainly all important data. Guidance like the 3-2-1 rule suggest keeping multiple copies, across multiple media and multiple sites. This will increase the carbon cost of storage accordingly, and should be accounted for.
+
+Removable media may also be considered for long-term storage. Their intermittent use makes it harder estimate energy use while active, though published data imply that energy use is significantly lower than that for fixed drives. Their embedded carbon may still be measured:
+
+| Storage Type       | Embedded Carbon (kgCO2e) | Media Embedded Carbon (kgCO2e/TB) |
+| ------------------ | -----------------------: | --------------------------------: |
+| Tape Drive (LTO 8) | 46.6                     | 0.62                              |
+
+### Indirect
+
+{% include categoryLabel.html label="CatC" %}
+
+{% include categoryItem.html item="CatCManaged" id="Managed" %}
+
+Much data is now stored on cloud storage - that is, storage managed by a cloud provider and accessed over the internet. Almost all cloud storage exists, ultimately, in HDD or SSD, and so while embedded carbon costs are hidden from the user, they are still incurred by the provider.
+
+Cloud storage provision also incurs a carbon cost from use, and this can be more visible to the end user. Some cloud providers allow users to examine estimated emissions from their services, but not all do; still, emissions can be estimated since they are roughly proportional to the amount of data stored. Storing one terabyte of data on a major cloud storage provider for one year uses around 35 kilowatt-hours of electricity.
```

Comment:
Wondering if there is a source for the 35 kWh of electricity or how it was calculated?